### ISSUE TITLE
UnsatisfiedLinkError on some devices

### ISSUE THREAD

**Issue by harunurhan** [ragnraok/android-image-filter#14]
I added this library to my projects, it works properly on some devices but it does not on some others. 
System.loadLibrary line, throws an exception because library cannot be loaded. I unzip the apk and check libAndroidImageFilter.so is created for all architecture.

There is no common among problematic devices. Some of them are KitKat whereas some other are JellyBean. Most of them are Samsung but there are also several HTC. Also there are a Samsung devices which does not have any crash. Furthermore, even though all of them has armv7 based cpu, there are a few armv7 based devices on which the library is able to run.

In short, I have no idea why it crashes on some devices.
